libtelemetry/src/main/java/com/mapbox/android/telemetry/TelemetryClient.java

```diff
@@ -50,6 +53,40 @@ void sendEvents(List<Event> events, Callback callback) {
     sendBatch(batch, callback);
   }
 
+  void sendAttachment(Attachment attachment, Callback callback) {
```

Comment:
What are some ways we can improve readability of this method? Can we split the content of it into a few helper methods?